knowledge-base/configuring-the-image-rendering-extension-to-generate-tiff-files.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Configuring the Image Rendering Extension to generate TIFF files
+description: Editing the Image Rendering Extension to render TIFF files.
```

Comment:
Based on the KB's content you can add more keywords about bigger/smaller size images/resolution and files as a whole. For example, the demo code in the solution explains how to "modify the original **Image** rendering extension settings to generate high resolution (300 x 300 dpi) images and register four new formats that use the **CCITT G3/G4**, **RLE** and **None** compressions." This must be mentioned in the description.